metadata.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "theforeman-foreman_network",
+  "version": "0.1.0",
+  "author": "theforeman",
+  "summary": "network configuration",
+  "license": "GPLv3+",
+  "source": "git://github.com/theforeman/puppet-foreman_network",
+  "project_page": "https://github.com/theforeman/puppet-foreman_network",
+  "issues_url": "https://github.com/theforeman/puppet-foreman_network/issues",
+  "description": "Module for configuring network, firewall and ntpd",
+  "tags": [
+    "foreman",
+    "network"
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 2.7.0 < 4.0.0"
+    }
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Fedora",
+      "operatingsystemrelease": [
+        "19"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "12.04",
+        "14.04"
+      ]
+    }
+  ],
+  "dependencies": [
+    {"name":"puppetlabs/stdlib","version_requirement":">= 2.3.0"},
+    {"name":"razorsedge/network","version_requirement":">= 2.0.0"}
```

Comment:
@domcleal, would  you find example42/network to be better option? https://forge.puppetlabs.com/example42/network